homework/index.js

```diff
@@ -1,54 +1,81 @@
 'use strict';
 
-{
```

Comment:
You have removed the curly braces which surround all the code. Without these braces (that create a block scope), all your functions are now "polluting" the global scope. It is best to keep them of global scope.